installation.html

```diff
@@ -11,312 +11,216 @@
+        the BrowserSync UI)
     </li>
-  </ul>
-</p>
-<p>
-  <code>
-    docker run -d -p 8080:8080 -p 3000:3000 -p 3001:3001 -p 4022:22 -t jdubois/jhipster-docker
-  </code>
-</p>
-<p>
-  <br/>Make a volume container
-</p>
-<p>
-  The Docker <b>-v</b> option should be used to share the Docker "/jhipster" folder to the local "~/jhipster" folder.
-  However, this option doesn't work on MAC (issue <a href="https://github.com/docker/docker/issues/4023">4023 on docker project</a>).
-  The workaround is to use the <a href="https://github.com/SvenDowideit/dockerfiles/tree/master/samba">svendowideit/samba</a> project.
-  More information can be found on the <a href="Folder sharing">https://github.com/boot2docker/boot2docker#folder-sharing</a> section on the boot2docker website.
-</p>
-<ol>
-  <li>
-    <p>
-      Retrieve the name of the container
-    </p>
-    <p>
-      Here is the resultat of the command `docker ps`. Get the name at the column NAMES.
-      <code>
-      CONTAINER ID        IMAGE                            COMMAND                CREATED             STATUS              PORTS                                                                                            NAMES<br/>
-      418448ff3371        jdubois/jhipster-docker:latest   "/bin/sh -c '/usr/sb   59 minutes ago      Up 59 minutes       0.0.0.0:4022->22/tcp, 0.0.0.0:8080->8080/tcp, 0.0.0.0:3000->3000/tcp, 0.0.0.0:3001->3001/tcp   <b>naughty_bartik</b>
-      </code>
-    </p>
-  </li>
-  <li>
-    <p>
-      Add the samba support.
-    </p>
-    <p>
-      <code>
-        docker run --rm -v /usr/local/bin/docker:/docker -v /var/run/docker.sock:/docker.sock svendowideit/samba <b>naughty_bartik</b>
-      </code>
-    </p>
-  </li>
-  <li>
-    <p>
-      Folder sharing
-    </p>
-    <p>
-      <code>
-        goto Go|Connect to Server in Finder <br/>
-        enter 'cifs://192.168.59.103 <br/>
-        hit the 'Connect' button <br/>
-        select the volumes you want to mount <br/>
-        choose the 'Guest' radio button and connect <br/>
-      </code>
-    </p>
-  </li>
-  <li>
-    <p>
-      Once mounted, will appear as /Volumes/jhipster. Create a symlink to make the volume accessible in the local "~/jhipster" folder.
-    </p>
-    <p>
-      <code>
-        ln -s /Volumes/jhipster ~/jhipster
-      </code>
-    </p>
-  </li>
-</ol>
-</p>
-<h3>SSH configuration</h3>
-<p>
-You can now connect to your docker container with SSH. You can connect as "root/jhipster" or as "jhipster/jhipster", and we recommand you use the "jhipster" user as some of the tool used are not meant to be run by the root user.
-</p>
-<p>
-Start by adding your SSH public key to the Docker container:
+</ul>
 </p>
 <p>
-<code>
-cat ~/.ssh/id_rsa.pub | ssh -p 4022 jhipster@localhost 'mkdir ~/.ssh && cat >> ~/.ssh/authorized_keys'
-</code>
-</p>
+    <code>
+        docker run --name jhipster -v ~/jhipster:/home/jhipster/app:rw -v ~/.m2:/home/jhipster/.m2:rw -p 8080:8080 -p
```

Comment:
Add the working dir :
```
docker run --name jhipster -w /home/jhipster/app -v ~/jhipster:/home/jhipster/app:rw -v ~/.m2:/home/jhipster/.m2:rw -p 8080:8080 -p 3000:3000 -p 3001:3001 -d -t jdubois/jhipster-docker
```

The user will be directly in the good directory, when he goes inside container